app/main.py

```diff
@@ -1,3 +1,68 @@
+from typing import List, Dict, Any
+
+
+class Knight:
+    def __init__(self,
```

Comment:
After opening the parenthesis, write all the arguments on a new line, including the first one